lib/engine.js

```diff
@@ -457,7 +457,7 @@ module.exports = function container (get, set, clear) {
                     return cb(err)
                   }
                 }
-                if (n(s.balance.asset).subtract(s.balance.asset_hold || 0).value() < n(size).value()) {
+                if (n(s.balance.asset || 0).value() < n(s.balance.asset_hold).value()) {
```

Comment:
Unfortunately, I am unsure of what to do with this line. I'm unsure of the mathematical error when the output is correct. 0.0% and 0.000000 on hold. I am not very familiar with how other exchanges or the rest of the code computes into this line and feel like it is beyond my scope right now. I do know that this seems to work 100% for me for some odd reason. I never actually see "order size" since I'm always buying and selling 100% of balance every order value. Such as, if I'm selling I'm selling 100% of my balance at once in one order. If that makes any sense. This is why taker mode works so great on poloniex. Because you need to place an order for 100% of your altcoin balance in order for it to enguage. IDK, let me think about what I can do, I'll need to dig into the code and test with kraken as well to see what's going, any help is appreciated and I appreciate your pointing me in the right direction 👍 :)